variables.js

```diff
@@ -0,0 +1,3 @@
+module.exports = async function (self) {
+	self.setVariableDefinitions([])
```

Comment:
You are not defining your variable tcp_response, it will be unusable

Reply:
removed the variable from the TCP section -- it was not being used